net/oor/Makefile

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2011, 2015 Cisco Systems, Inc.
+# Copyright (C) 2015 CBA research group, Technical University of Catalonia.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=oor
+PKG_REV:=e74e35daa23c4993b2b38e4ac3e867526bc65e70
+PKG_VERSION:=1.2.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=git://github.com/OpenOverlayRouter/oor.git
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=$(PKG_REV)
+
```

Comment:
Please codeload instead or preferably upload a proper tarball to GitHub.
Example of codeload: https://github.com/openwrt/packages/blob/master/sound/fdk-aac/Makefile#L15

Reply:
I have updated the makefile to use the the tarball instead of git and fixed the other requests but I have never used cmake. If it is mandatory to move to cmake, can you point me to an example or guide in order I can migrate it more easily?